index.js

```diff
@@ -1,44 +1,32 @@
-'use strict';
+var path = require('path');
 
 var express = require('express');
 var favicon = require('serve-favicon');
-var path = require('path');
+
 
 module.exports = function(app, baseDir, config) {
-    var env = app.get('env');
     var servedDirectory = 'app';
 
-    if ('development' === env) {
-        // Disable caching of scripts for easier testing
-        app.use(function noCache(req, res, next) {
-            if (req.url.indexOf('/js/') === 0) {
-                res.header('Cache-Control', 'no-cache, no-store, must-revalidate');
-                res.header('Pragma', 'no-cache');
-                res.header('Expires', 0);
-            }
-            next();
-        });
-
-        app.use(config.baseUrl, express.static(path.join(__dirname, '.tmp')));
-    }
+    if ('production' === app.get('env')) {
+        servedDirectory = 'dist';
 
-    if ('production' === env) {
         // prerender.io
         app.use(require('prerender-node').set('prerenderToken', process.env.PRERENDER_TOKEN).set('protocol', 'https'));
-
-        servedDirectory = 'dist';
-        app.use(favicon(path.join(__dirname, 'dist', 'favicon.ico')));
+    } else {
+        app.use(config.baseUrl + '/styles', express.static(path.join(__dirname, '.tmp', 'styles')));  // ugly hack to serve compiled SCSS; you will need to `grunt build` every time you change a style
     }
 
-    app.use(config.baseUrl, express.static(path.join(__dirname, servedDirectory)));
+    servedDirectory = path.join(__dirname, servedDirectory);
+
+
+    app.use(config.baseUrl, express.static(servedDirectory));
     app.use(config.baseUrl + '/scripts/template.js', express.static(path.join(baseDir, config.scenarioTemplate)));
     app.use(config.baseUrl + '/scripts/constants.js', express.static(path.join(baseDir, config.constants)));
-
-    app.get(config.baseUrl + '/scripts/server-config.js', function (req, res) {
+    app.get(config.baseUrl + '/scripts/server-config.js', function(req, res) {
```

Comment:
I use [Douglas Crockford](http://javascript.crockford.com/code.html#variable%20declarations) conventions for JavaScript since some months.

> If a function literal is anonymous, there should be one space between the word function and the ( (left parenthesis). If > the space is omited, then it can appear that the function's name is function, which is an incorrect reading.

But we are not consistent in many projects...